docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: "Device Trust (Preview)"
+description: Learn how to use and enforce trusted devices with Teleport
+---
+
+<Admonition type="warning" title="Preview">
+  Device Trust is currently in Preview mode.
+</Admonition>
+
+<ScopedBlock scope="oss">
+  Device Trust requires a commercial edition of Teleport.
+</ScopedBlock>
+
+Device Trust allows Teleport admins to enforce the use of trusted devices.
+Resources protected by the device mode "required" will enforce the use of a
+trusted device, in addition to establishing the user's identity and enforcing
+the necessary roles. Furthermore, users using a trusted device leave audit
+trails that include the device's information.
+
+The device trust preview supports the following Teleport features:
+
+- SSH access enforcement
+- Database access enforcement
+- Kubernetes access enforcement
+- macOS trusted devices
```

Comment:
The last one is more of a system requirement than a Teleport feature.

I would say "the device trust preview requires macOS and supports the following Teleport features:"

Reply:
Rephrased as "The device trust preview works only with macOS devices and supports the following Teleport features". Let me know if you prefer your wording.